smoke/features/data/shareconfigmap.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
+set -x
+
+create_shared_configmap(){
+  oc create -f - <<EOF
+  kind: SharedConfigMap
+  apiVersion: sharedresource.openshift.io/v1alpha1
+  metadata:
+    name: my-shared-config
+  spec:
+    configMapRef:
+      namespace: $1
+      name: share-config
+EOF
+}
+
+create_rbac(){
+  oc create -f - << EOF
+  apiVersion: rbac.authorization.k8s.io/v1
+  kind: Role
+  metadata:
+    name: shared-resource-my-shared
+    namespace: $1
+  rules:
+    - apiGroups:
+        - sharedresource.openshift.io
+      resources:
+        - sharedconfigmaps
+      resourceNames:
+        - my-shared-config
+      verbs:
+        - use
+EOF
```

Comment:
should you not also be creating a rolebinding that maps this role to the `default` serviceaccount?   I'm not seeing that being handled anywhere else

Reply:
Ohh yes, I missed it. I realised it after pushing the commit. So, I planned to push it in my next commit 🙂 . Thanks for pointing it as well.